WordPress/wordpress-playground
# [CLI] Fix `isWordPressInstalled()` in CLI by inlining the auto_login.php in index.ts instead of using import ?raw

## Motivation for the change, related issues

[WordPress Major and Beta update workflow](https://github.com/WordPress/wordpress-playground/actions/workflows/refresh-wordpress-major-and-beta.yml) is currently failing because the CLI can't start. We've introduced a bug in https://github.com/WordPress/wordpress-playground/pull/1856 by adding [a `raw` import to the `@wp-playground/wordpress` package](https://github.com/WordPress/wordpress-playground/pull/1856/files#diff-8afd615c3847003caebf2b35102f3a2a107ad9647e095495ecc9302862802191R7).

Raw imports aren't native to JavaScript, they are processed on build, using Vite in our case.
This is why raw imports work well in our Remote package, but not in the CLI. 

When Playground is started using Node, the [CLI tries getting the raw text from `auto_login.php`](https://github.com/WordPress/wordpress-playground/pull/1856/files#diff-8afd615c3847003caebf2b35102f3a2a107ad9647e095495ecc9302862802191R7).
Instead of getting the text, it gets the path to the imported file (e.g. `/Users/me/Projects/wordpress-playground/packages/playground/wordpress/src/mu-plugins/auto_login.php`).
This path is written into the `1-auto-login.php` mu-plugin as a string.

When Playground tries to boot, it [checks if WordPress is installed](https://github.com/WordPress/wordpress-playground/blob/ccadc7d0daedda0b59b571e12aade32cf3abaeab/packages/playground/wordpress/src/boot.ts#L243). 
The expected result of that check is `1`, but because  `1-auto-login.php` contains the path as a string, and because WordPress uses PHP `include` to load that file, the path gets written to stdout and mixed with the script output. The outcome is that `result.text` is `/Users/me/Projects/wordpress-playground/packages/playground/wordpress/src/mu-plugins/auto_login.php1` instead of `1`.

## Implementation details

This PR moves the auto-login code from a PHP file to inline code.

The approach matches what we did with other `mu-plugins` that must be loaded both in the CLI and on the web.

TODO:
- [Add tests to ensure the CLI works](https://github.com/WordPress/wordpress-playground/issues/1871)
- Consider adding instructions for setting up the CLI as a global package

## Testing Instructions (or ideally a Blueprint)

- Run `bun packages/playground/wordpress-builds/build/build.js --wp-version=latest-minus-1 --output-js=packages/playground/wordpress-builds/src/wordpress --output-assets=packages/playground/wordpress-builds/public`
- Confirm it works
